src/routes/webhook/webhook.service.spec.ts

```diff
@@ -72,18 +82,18 @@ describe('Webhook service', () => {
         .spyOn(webhookService, 'postWebhook')
         .mockImplementation(async () => postWebhookResponse);
 
-      const msg = { text: 'hello' };
+      const msg = {
+        chainId: '1',
+        type: 'SAFE_CREATED' as TxServiceEventType,
+        text: 'hello',
+        address: '0x0275FC2adfF11270F3EcC4D2F7Aa0a9784601Ca6',
+      };
       const results = await webhookService.postEveryWebhook(msg);
       expect(results).toEqual([postWebhookResponse, postWebhookResponse]);
```

Comment:
If we have three different webhooks with three diferent urls shouldn't be three results?

Reply:
One of them has

`webhooks[1].sendSafeCreations = false;`

so webhook will not be called for this message. I'm adding a comment